cmd/policy/README.md

```diff
@@ -0,0 +1,50 @@
+## Example usage
+
+    ./policy list
+
+List all policies in Veraison store. This will output a list of tenant ID and
+format name pairs.
+
+    ./policy -t 2 list
+
+Same as above but only list policies for tenant ID 2.
+
+    ./policy add -f my_policies.zip
+
+Add policies from my_policies.zip for tenant 1, overwriting any existing policies.
+
+   ./policy -c /opt/veraison/config/ get -t 3 -o my_policies.zip psa dice
+
+Get policies for psa and dice token formats for tenant 3, using Veraison
+configuration from /opt/veraison/config/, and write them to my_policies.zip.
+
+ ./policy verify -e test/endorsements.json test/evidence-context.json
+
+Run stored policy against input inside test/evidence-context.json, using
```

Comment:
Which policies are used in this case?  Is there a default tenant or all applies?

Reply:
Defaults to tenant 1. I will update the docs to make that clearer. Thanks!